docs/cli_getting_started.md

```diff
@@ -0,0 +1,112 @@
+# Getting started with the Kili CLI
+
+## What is the Kili CLI
+
+The Kili CLI has been designed to run key actions on your projects with one line of code.
+For the supported actions, the CLI offers a more compact way to manage your projects than the Python SDK. However the Python SDK offers more possibilities and must be used for more complex project management tasks.
+
+## Authentification
+
+- Create and copy a [Kili API key](https://docs.kili-technology.com/docs/creating-an-api-key)
+- Add the `KILI_API_KEY` variable in your bash environment (or in the settings of your favorite IDE) by pasting the API key value you copied above:
+
+  ```bash
+  export KILI_API_KEY='<you api key value here>'
+  ```
+
+!!! info
+    While launching commands, you can also inform you API key through the `--api-key` option. If have set your `KILI_API_KEY` environement variable and provide an API key through the `--api-key` option, it will used the one from the command's option.
+
+## Usage
+
+The main command is `kili`. It currently only have one subcommand `project` that gathers all the commands for project management :
+
+```
+kili project [COMMAND]
+```
+
+## Workflow example
+
+Let's take an example where you want to start a project from scratch:
+
+### Create a project
+
+To create an IMAGE project:
+
+```
+kili project create \\
+            --interface path/to/interface.json \\
+            --input-type IMAGE \\
+            --title "Defect detection"
+            --description "Bottle defects on production line"
+```
+
+Returns:
+
+```
+Project successfully created. Id: cl4ljd3awc5gj0lpbb89nbcqg'
+```
+
+### List your projects
+
+```
+kili project list --max 10
+```
+
+Returns:
+
+```
+title                                     id                           progress  description
+----------------------------------------  -------------------------  ----------  -------------------------------
+Defect detection                          cl4ljd3awc5gj0lpbb89nbcqg        0.0%  Bottle defects on production...
+invoice NER                               cl3d43bzb0rl71mx4580mpbt1       92.5%  For intelligent document pro...
+```
+
+### import data to your project
+
+To import data, you can provide a list of files or folders
+
+```
+kili project import \\
+    reference_image.png datasets/defect_detection/ \\
+    --project-id cl4ljd3awc5gj0lpbb89nbcqg \\
+```
+
+Returns:
+
+```
+Files skipped: [datasets/defect_detection/visit1.mp4, datasets/defect_detection/visit2.mp4]. Paths either do not exist, are filtered out or point towards wrong data type for the project
```

Comment:
Note for a further change (other PR needed):
 the default output should only say `2 files have been skipped`. Passing --verbose or --file-list should return something like:
```
datasets/defect_detection/visit1.mp4 SKIPPED
datasets/defect_detection/visit2.mp4 SKIPPED
...
```
so it can be parsed by a downstream job

Reply:
completely agree! I create the ticket